requirements.txt

```diff
@@ -1,6 +1,3 @@
 numpy
 scipy
 scikit-learn
-pep8
-pyflakes
-nose
```

Comment:
I'm not sure what you mean here. Do you mean things like `nose.tools.assert_equal`? `nose` is still a test dependency, it has just moved to be listed in the `requirements-test.txt` file and the `setup.py` file.

If you are concerned about a `NameError` on the build tests, these dependencies have moved to `requirements-test.txt`, so they are installed by `pip` in the `.travis.yml` file. If you are concerned about a user running `python setup.py test` without installing these dependencies, `nose` will be installed automatically there because it is now listed in `tests_require` in `setup.py`.